source/catalog.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+
+<body class="page">
+
+  <header class="main-header">
+    <a class="main-header__logo logo">
+      <img class="logo__image" src="img/logo-image.svg" alt="Логотип Кэт энерджи" width="60" height="50">
+      <img class="logo__text" src="img/logo-text.svg" alt="Кэт энерджи">
+    </a>
+    <nav class="main-nav">
+      <ul class="main-nav__list site-list">
+        <li class="site-list__item">
+          <a class="site-list__navigation" href="index.html">
+            главная
+          </a>
+        </li>
+        <li class="site-list__item">
+          <span class="site-list__navigation site-list__item--active">
```

Comment:
```suggestion
          <span class="site-list__navigation site-list__navigation--active">
```

Тут тоже модификатор идёт не со своим элементом, и [Йоксель](https://yoksel.github.io/html-tree/) ругается